CHANGELOG.md

```diff
@@ -214,6 +214,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Add man pages and command reference to userdocs. #1488
 - Document building images from scratch with Apptainer. #1485
 - Added warewulfd:/overlay-file/{overlay}/{path...}?render={id}
+- Add `wwct configure rsync` and the template `rsyncd.conf.ww`
+- Make dracut aware of `root==perisitent` which will install the node image to the parition `rootfs`
```

Comment:
I think I said this before; but `root=persistent` is, imo, too generic. I already think the current `root=wwinit` is too generic; maybe `root=wwinit:tmpfs` would be better, then maybe this could be `root=wwinit:ignition` which would leave room for `root=wwinit:cloud-init` in the future.